test/jsonapi_test.exs

```diff
@@ -9,7 +9,7 @@ defmodule JSONAPITest do
     def type, do: "mytype"
     def relationships do
       [author: {JSONAPITest.UserView, :include},
-       other_user: {JSONAPITest.UserView, :include}]
+       other_user: JSONAPITest.UserView]
```

Comment:
Will these changes result in a breaking change?  If not, let's leave the existing tests alone and add new ones to cover these changes.

Reply:
Yeah this changes the functionality. Although, I think it fixes the original intended functionality of `:include` meaning that the relationship will be included by default. I'm basing this on this bit from the readme:

```
  def relationships do
    # The post's author will be included by default
    [author: {MyApp.UserView, :include},
     comments: MyApp.CommentView]
  end
```

Thoughts?

Reply:
@doomspork @jeregrine ping